application/src/main/java/com/sanction/thunder/resources/VerificationResource.java

```diff
@@ -28,9 +34,10 @@
   private static final Logger LOG = LoggerFactory.getLogger(VerificationResource.class);
 
   private final PilotUsersDao usersDao;
+  private final EmailService emailService;
 
   // Counts number of requests
-  private final Meter postRequests;
+  private final Meter getRequests;
```

Comment:
Let's rename this to something better. Make two `Meters` one for `userVerificationRequests` and one for `emailVerificationRequests` or something like that.